style.css

```diff
@@ -137,21 +201,23 @@ button {
   border-radius: 5px;
   background-color: white;
   padding: 10px;
+  width: fit-content;
 }
 
 .project button {
   display: grid;
-  margin: 30px 160px 30px 0;
+  margin: 30px 160px 10px 0;
```

Comment:
- [ Optional ] I don't think you need a grid container for the button only. Since the button is Flex item, you can use Flexbox properties to add alignments. please, take a look at [this](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) website various options in the Flexbox.

Reply:
Done. I don't remember doing this 😂 
Must have been by mistake.